### PR TITLE
Binder runnable

### PR DESCRIPTION
**Pull request by dr3y**
i added requirements.txt and i put in the try except so that the default version of dnaplotlib can be used

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+fa2
+lxml
+beautifulsoup4
+cython
+sympy
+pandas
+astropy
+emcee
+python-libsbml
+#--editable git+git://github.com/biocircuits/bioscrape.git@master#egg=bioscrape
```
Comment (zoltuz) [inline requirements.txt:14]: do you want to remove this for now?

Comment (dr3y) [reply requirements.txt:14]: ok let me remove it and make sure the binder still works. it should though

Comment (dr3y) [reply requirements.txt:14]: i tried to make it load bioscrape obviously that didnt work

Comment (zoltuz) [reply requirements.txt]: I figured, but let's not accumulate dead code.

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+--editable git+git://github.com/dr3y/dnaplotlib.git@master#egg=dnaplotlib
+bokeh
+matplotlib
+networkx
+fa2
+lxml
+beautifulsoup4
```
Comment (zoltuz) [inline requirements.txt]: do we need this?

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+--editable git+git://github.com/dr3y/dnaplotlib.git@master#egg=dnaplotlib
+bokeh
+matplotlib
+networkx
+fa2
+lxml
```
Comment (zoltuz) [inline requirements.txt]: and this one?

Comment (dr3y) [reply requirements.txt]: i think i left in only the essentials

Comment (dr3y) [reply requirements.txt]: the binder is loading...

Comment (dr3y) [reply requirements.txt]: ok it works

**Pull request merged**